snc.sh

```diff
@@ -43,20 +43,14 @@ else
     fi
 fi
 
-# Download the oc binary for all platforms
-mkdir -p openshift-clients/linux openshift-clients/mac openshift-clients/windows
-if [[ ${OKD_VERSION} != "none" ]]
-then
-    curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-linux-${OPENSHIFT_RELEASE_VERSION}.tar.gz" | tar -zx -C openshift-clients/linux oc
-    curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-mac-${OPENSHIFT_RELEASE_VERSION}.tar.gz" | tar -zx -C openshift-clients/mac oc
-    curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-windows-${OPENSHIFT_RELEASE_VERSION}.zip" > openshift-clients/windows/oc.zip
+# Download the oc binary for the specific architecture
+download_oc_binary
+if [ "$?" -eq 0 ]; then
+        OC=./openshift-clients/linux/oc
```

Comment:
`set -exuo pipefail` at the beginning of the script will make the script abort if there's an error, so you should not need to check the exit code. Is it possible to set `OC` in `download_oc_binary`? Is it desirable?